.github/workflows/install-shared-dependencies/action.yml

```diff
@@ -48,12 +49,24 @@ runs:
           run: |
               yum install -y gcc pkgconfig openssl openssl-devel which curl redis6 gettext --allowerasing
 
+        - name: Install software dependencies for Ubuntu MUSL
+          shell: sh
+          if: "${{ inputs.os == 'ubuntu-latest-musl' }}"
+          run: |
+              apk update
+              wget -O - https://sh.rustup.rs | sh -s -- -y
+              source "$HOME/.cargo/env"
+              apk add protobuf-dev musl-dev make gcc
```

Comment:
Don't you need to install openssl-dev?

Reply:
`openssl-dev` is installed as part of the `musl-dev` .
Do you suggest to explicitly specify it?